frontend/riskofbias/constants.js

```diff
@@ -77,6 +77,19 @@ const NA_KEYS = [10, 20],
         26: "Adequate (metric) or Medium confidence (overall)",
         27: "Good (metric) or High confidence (overall)",
     },
+    BIAS_DIRECTION_UNKNOWN = 0,
+    BIAS_DIRECTION_UP = 1,
+    BIAS_DIRECTION_DOWN = 2,
+    BIAS_DIRECTION_SIMPLE = {
+        0: "?",
```

Comment:
for this and other fields, can you change the 0 to our new constant BIAS_DIRECTION_UNKNOWN